ogc/src/main/scala/geotrellis/server/ogc/OgcTime.scala

```diff
@@ -110,6 +150,19 @@ object OgcTimePositions {
   *                       encoded directly
   */
 final case class OgcTimeInterval(start: ZonedDateTime, end: ZonedDateTime, interval: Option[String]) extends OgcTime {
+  def duration: Option[Duration] = interval.flatMap(p => Try(Duration.parse(p)).toOption)
```

Comment:
It is a Duration in terms of `ISO`, but it should be Period+Duration in terms of Java Time. Check out this SO link: https://stackoverflow.com/questions/46258478/parsing-the-iso-8601-duration-values-of-the-amazon-duration-slot-type 

I would bring a new `org.threeten.extra.PeriodDuration` dep into the project that parses ISO Duration correctly.

cc @jisantuc